packages/serve/src/createConfig.ts

```diff
@@ -1,4 +1,8 @@
 import { devServerOptionsType } from './types';
+// eslint-disable-next-line node/no-extraneous-import
+import { version } from 'webpack-dev-server/package.json';
```

Comment:
We should use `packageExist` or use `try`/`catch`

Reply:
yep on it

Reply:
handled at the entry of serve package to avoid redundant checks